### ISSUE TITLE
`render_all` action not available during playback (animation needs to be paused before being rendered)

### ISSUE THREAD

**Issue by mgpjm** [coldtype/coldtype#94]
Hi Rob, I was trying to export frames from an animation but the shortcut "a" doesn't seems to work. No messages appear on the terminal, just it seems to not take the command. I'm doing that when the animation window is active, and it seems to take other shortcuts.
Also, if I try to export a GIF (witch is my real purpose) I got this message
```
Could find no file with path '/Users/mgpjm/RVVV/animations/renders/animation6/recursive/recursive_%4d.png' and index in the range 0-4
/Users/mgpjm/RVVV/animations/renders/animation6/recursive/recursive_%4d.png: No such file or directory
```
I used this code 
```
def release(passes):
    FFMPEGExport(recursive, passes).gif().write()
```
What I'm doing wrong? I run Coldtype on Mac M1 Monterey.

Thank you
m

**Comment by stenson** [comment]
Hmm, yeah the gif won't work until the individual frames are rendered to disk, so the problem is definitely with the rendering part, i.e. the `a` shortcut. Are you hitting `a` while the viewer app is focused in the foreground?

You could also try a "headless" render, by running the coldtype command with an `-a` appended, to see if that prints any errors.

**Comment by mgpjm** [comment]
> You could also try a "headless" render, by running the coldtype command with an -a appended, to see if that prints any errors.
Ok, it work out in this way! However, it is strange that the shortcut does not work, no?
Also, there is a  "headless" way to export directly a gif?  Otherwise I'll put my frames in After and make a gif from there.

Thank you so much!

**Comment by stenson** [comment]
Good to hear! Though yes, definitely strange that the shortcut does not work. Do any shortcuts work in the viewer? For instance, if you hit just `q` does the viewer exit and quit?

**Comment by stenson** [comment]
Here's the same video embedded on github (wanted to see if this would work)

https://user-images.githubusercontent.com/89770/158094900-681fc7a1-2719-45cc-881e-b6d9ffe34a0d.mov

**Comment by mgpjm** [comment]
`Do any shortcuts work in the viewer? For instance, if you hit just q does the viewer exit and quit?`
Yep, any other shortcut seems to work, also `q`

I tried `render all` option and it works too!

I having some trouble make it works the `release` command. It works once, I don't really understand how, because I have to defined the plus name

**Comment by stenson** [comment]
Sorry for the delay in response here — I was just talking with a friend about Coldtype, though, and they were having the same problem with the `a` shortcut — by any chance is the animation/playback running when you hit the `a` key? (If so you'll have to pause the animation first before hitting the a key. This is a bug so hopefully I'll be able to fix it in the future.)

**Comment by mgpjm** [comment]
> (If so you'll have to pause the animation first before hitting the a key. This is a bug so hopefully I'll be able to fix it in the future.)

Oh my, I tried and is actually working. I swear I was just sure I tried already! Well, anyway, I hope I helped a little bit on the debugging part :)

Thank you so much!
m

**Comment by stenson** [comment]
(fix will be available in 0.9.8)
